src/components/Actions.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Actions = () => {
+  return (
+    <div>
+      <div className="actions">
+        {/* Font Awesome icons */}
+        <i class="far fa-comment"></i>
```

Comment:
tanto en javascript como en react class es una palabra reservada entonces no podemos hacer uso de ella para clases de css entonces hay que cambiar todas a className para referirnos a esas class de css 👀 
```suggestion
        <i className="far fa-comment"></i>
          <i className="fas fa-retweet"></i>
        <i className="far fa-heart"></i>
        <i className="fas fa-share"></i>
```